start.js

```diff
@@ -89,10 +92,24 @@ $(function(){
         }
         $("audio").volume = vol;
         $("audio").prop("volume", vol);
-        $("#vol_val").text(vol);
-        
+        $("#vol_val").text(vol_origin);
     })
 
+    //스피커 이미지 선택시 볼륨 0으로, 다시 누르면 원래 볼륨으로
+    $("#speakerImg").on ("click", function() {
+        if($("#volume-bar").prop("value") == 0)
+        {
+            $("#volume-bar").prop("value", volumTmp);
+            $("#volume-bar").trigger("change");
+            return;
+        }
+
+        volumTmp = $("#volume-bar").val();
+        
+        $("#volume-bar").prop("value", 0);
+        $("#volume-bar").trigger("change");
+	})
```

Comment:
`$("#volume-bar")`의 중복이 많은데 위와 같이 변수로 빼는 것도 괜찮아보여요.
추가로, `$("#volume-bar").prop("value")`은 `$("#volume-bar").val()`과 동일해서 한번 코멘트 남겨봅니다~
```suggestion
        const volumeBar = $("#volume-bar");
        let currentVolume = volumeBar.val();

        if (currentVolume == 0) {
            volumeBar.val(volumTmp).trigger("change");
            return;
        }

        volumTmp = currentVolume;
        volumeBar.val(0).trigger("change");
```